builders/buildPost.ts

```diff
@@ -51,5 +53,27 @@ const buildContent = (
   const images =
     imgNodes && imgNodes.toArray().map(element => element.attribs["src"]);
 
-  return { message, images };
+  // fetch any external links
+  let postLink: PostLink | undefined = undefined;
```

Comment:
This method is doing a bit too much already. What do you think about extracting `buildMessage`, `buildMessage`, `buildPostLink` methods? They could accept `userContent` as an only argument.

Reply:
I think we can make this a separate task, not only to split that method but also to refactor post scrapper (eq. by adding post types - text, video, link, image etc)

Reply:
:+1 ok, just please create a ticket for this before merging this PR. thanks in advance.

Reply:
Needs to be discussed  #13